openinghours.go

```diff
@@ -233,54 +342,7 @@ func getWeekDay(weekday weekInt) string {
 	}
 }
 
-// GetWeekInt converts a weekday name (case-insensitive) to its corresponding RFC 3339 complient integer (1-7)
-// or returns 0 for invalid input.
-func GetWeekInt(v string) int {
-	switch strings.ToLower(v) {
-	case "monday":
-		return 1
-	case "tuesday":
-		return 2
-	case "wednesday":
-		return 3
-	case "thursday":
-		return 4
-	case "friday":
-		return 5
-	case "saturday":
-		return 6
-	case "sunday":
-		return 7
-	default:
-		return 0
-	}
-}
-
-// getHumanReadableTime parses a time string and returns the corresponding weekday, time, and week integer.
-// The `v` parameter is the time string to parse, which should follow the pattern "W{week}T{hour}:{minute}:{second}".
-// The `endTime` flag determines if the time at the end of a period should adjust to "24:00" of previous weekday if "00:00".
-// Returns the week integer, friendly weekday name, formatted time string, and an error if parsing fails.
-func getHumanReadableTime(v string, endTime bool) (weekInt, string, string, error) {
-	re := regexp.MustCompile(`^W(\d)T(\d{2}):(\d{2}):\d{2}$`)
-	matches := re.FindStringSubmatch(v)
-	if len(matches) < 2 {
-		return 0, "", "", fmt.Errorf("invalid value `%s`", v)
-	}
-	weekIndex, _ := strconv.Atoi(matches[1])
-	weekI := weekInt(weekIndex)
-	hours := matches[2]
-	minutes := matches[3]
-	if hours == "00" && minutes == "00" && endTime {
-		weekI.Previous()
-		hours = "24"
-	}
-	weekday := getWeekDay(weekI)
-	time := fmt.Sprintf("%s:%s", hours, minutes)
-
-	return weekI, weekday, time, nil
-}
-
-func parseMinutesSinceMidnight(v1, v2 string) (int, error) {
+func parseminutesSinceMidnight(v1, v2 string) (int, error) {
```

Comment:
```suggestion
func parseMinutesSinceMidnight(v1, v2 string) (int, error) {
```

Reply:
(You'll tell me that GoLand refactors that for you automatically)